osu.Game/Screens/Edit/Timing/TimingSection.cs

```diff
@@ -35,9 +33,6 @@ protected override void OnControlPointChanged(ValueChangedEvent<TimingControlPoi
         {
             if (point.NewValue != null)
             {
-                bpmSlider.Current = point.NewValue.BeatLengthBindable;
-                bpmSlider.Current.BindValueChanged(_ => ChangeHandler?.SaveState());
```

Comment:
The removal of this line is changing behaviour. On master, if you change BPM via either the slider or the textbox and try to quit editor, it will prompt you whether the change should be saved. It won't do it anymore now that this `SaveState()` call has been removed. (And yes, it's probably a bit weird that this was attached to the _slider_ of all things, but alas.)

This _probably_ needs test coverage of some sort. There doesn't seem to be any right now, as far as I can tell.

Reply:
Oooh, I get it. I'll attach it to the textbox in a bit.
It was probably written like this because this textbox influences the slider which calls `SaveState()`